### PR TITLE
Fixed Footer, Navbar and refactored blogs fetch to redux

### PR DESCRIPTION
**Pull request by Hrushi2406**
1. Redesigned the Navbar
2. Fixed Footer alignment issue
I think the footer must be only on home page
3. Refactored the fetch code of the blog to redux

### CODE REVIEW

File: new_client/src/components/auth/Signin.jsx [hunk 1]
```diff
@@ -1,127 +1,127 @@
...
-						<Typography>
-							New User ? <Link to="/signup">Sign Up</Link> instead.
-						</Typography>
-					</Grid>
-				</Grid>
-			</Paper>
-		</Container>
-
-	)
+  const { isAuthenticated, error, login, clearErrors } = props;
```
Comment (pal-16) [inline new_client/src/components/auth/Signin.jsx:18]: Signin cant be accessed through any route.So we need to add that route from Signup

File: new_client/src/App.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-import Glimpse from './components/glimpses/Glimpse';
+import Header from "./components/Header";
+import Home from "./components/Home/home";
+import Footer from "./components/Footer";
+import About from "./components/pages/About";
+import Glimpse from "./components/glimpses/Glimpse";
 
-import Blogs from './components/blogs/Blog';
-import AddBlog from './components/blogs/AddBlog';
+import Blogs from "./components/blogs/Blog";
```
Comment (pal-16) [inline new_client/src/App.js:14]: Keeping uniformity by writing .jsx extension with all the files imported.

File: new_client/src/components/Footer.css [hunk 3]
```diff
@@ -1,46 +1,15 @@
 .footer {
-    float: left;
-    width: 100%;
-    background-color: #000000;
-    overflow: auto;
-    padding-left: 40px;
-  
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 2%;
```
Comment (pal-16) [inline new_client/src/components/Footer.css:4]: As shown in the photos show to you if we are keeping footer on all pages then we still need to align it

File: new_client/src/components/Footer.css [hunk 4]
```diff
@@ -1,46 +1,15 @@
...
+  background-color: #0d0d0d;
+}
```
Comment (pal-16) [inline new_client/src/components/Footer.css:6]: As shown in the photos ,if we are keeping footer on all pages then we still need to align it.

Comment (pal-16) [reply new_client/src/components/Footer.css:6]: image

Comment (pal-16) [reply new_client/src/components/Footer.css:6]: image

Comment (AliabbasMerchant) [reply new_client/src/components/Footer.css:6]: Right. @Hrushi2406 please do this change

File: new_client/src/App.js (hunk 2 above)
Comment (AliabbasMerchant) [reply new_client/src/App.js:14]: Keeping uniformity is right. But I believe lets instead remove the '.jsx' extension from the other imports.

Comment (pal-16) [reply new_client/src/App.js:14]: Yes,Okay.

**Comment by AliabbasMerchant** [comment]
@Hrushi2406  Wonderful! 💯  
A great job formatting the code and following conventions (Really liked that you made sure that we are using a single type of inverted-commas everywhere)  
@pal-16 and I have left a few comments.  
Please do the required changes. I will merge the PR then.

**Comment by ShubhankarKG** [comment]
Closing this, since this is, kind of, stale.

**Pull request closed without merging**